services/galley/test/integration/API/Teams/LegalHold.hs

```diff
@@ -785,18 +829,181 @@ testNoConsentBlockDeviceHandshake = do
   -- tracked here: https://wearezeta.atlassian.net/browse/SQSERVICES-454
   pure ()
 
-testNoConsentBlockOne2OneConv :: TestM ()
-testNoConsentBlockOne2OneConv = do
-  -- "If LH is activated for other user in 1:1 conv, 1:1 conv is blocked"
-  -- tracked here: https://wearezeta.atlassian.net/browse/SQSERVICES-429
-  pure ()
+-- If LH is activated for other user in 1:1 conv, 1:1 conv is blocked
+testNoConsentBlockOne2OneConv :: HasCallStack => Bool -> Bool -> Bool -> Bool -> TestM ()
```

Comment:
I admit it's a little hard to read this helper function as there are too many boolean branches here to figure out in one's head if this makes sense or not. I would personally prefer a bit more code duplication and less if/else branching on booleans.
This is a nitpick though and this can (or not) be done also as a follow-up cleanup.

Reply:
Stefan has paired excessively with me on this test, but I'll fix the boolean blindness in a separate PR anyway.